<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test3_carsClass</class>
 <widget class="QWidget" name="test3_carsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test3_cars</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>524</width>
     <height>286</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>All  cars: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="getAllSorted"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="gievManName">
        <property name="text">
         <string>Manufacurer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showcars">
        <property name="text">
         <string>Show cars</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="getallMan"/>
      </item>
      <item>
       <widget class="QLineEdit" name="giveNumberCars"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="test3_cars.qrc"/>
 </resources>
 <connections/>
</ui>
